stable/pachyderm/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+description: >
+  Pachyderm is a large-scale container-based workflow engine that offers complete version control for data and reproducible data processing
+name: pachyderm
+icon: https://avatars2.githubusercontent.com/u/10432478?s=200
+keywords:
+  - pachyderm
+  - workflows
+  - reproducibility
+  - distributed
+  - processing
+version: 0.1.0
+appVersion: 1.6.6
+home: "https://pachyderm.io"
+sources:
+  - "https://github.com/pachyderm/pachyderm"
+maintainers:
+  - name: Jon Ander Novella
```

Comment:
can you make sure all names are actually GitHub IDs? It makes it easier to reference back. Also, please add an OWNERS file (see [format](https://github.com/kubernetes/charts/blob/master/stable/drupal/OWNERS)), which will allow anyone listed there to approve PRs.